bal_tools/ecosystem.py

```diff
@@ -115,3 +120,82 @@ def get_aura_pid_from_gauge(self, deposit_gauge_address: str) -> int:
                     f"Got a list result with something other than 1 member when compiling aura PID mapping: {result}"
                 )
             return result[0]
+
```

Comment:
I did not review this file in detail but it is not in operation anywhere yet so we will see how it works